src/v1/publisher_client_config.json

```diff
@@ -2,23 +2,27 @@
   "interfaces": {
     "google.pubsub.v1.Publisher": {
       "retry_codes": {
+        "non_idempotent": [],
         "idempotent": [
-          "ABORTED",
-          "UNAVAILABLE",
-          "UNKNOWN"
+          "DEADLINE_EXCEEDED",
+          "UNAVAILABLE"
         ],
-        "non_idempotent": [
+        "unavailable": [
```

Comment:
In light of the default / retry harmonization efforts earlier, I think we should spend some manual time looking at the changes here. Making a note here for myself, partially.

Reply:
Okay, I see what's going on here, this should be fine.